lxd/storage/drivers/utils.go

```diff
@@ -537,21 +537,21 @@ func growFileSystem(fsType string, devPath string, vol Volume) error {
 	}
 
 	return vol.MountTask(func(mountPath string, op *operations.Operation) error {
-		var msg string
 		var err error
 		switch fsType {
 		case "ext4":
-			msg, err = shared.TryRunCommand("resize2fs", devPath)
+			_, err = shared.TryRunCommand("resize2fs", devPath)
 		case "xfs":
-			msg, err = shared.TryRunCommand("xfs_growfs", mountPath)
+			_, err = shared.TryRunCommand("xfs_growfs", mountPath)
 		case "btrfs":
-			msg, err = shared.TryRunCommand("btrfs", "filesystem", "resize", "max", mountPath)
+			_, err = shared.TryRunCommand("btrfs", "filesystem", "resize", "max", mountPath)
 		default:
 			return fmt.Errorf("Unrecognised filesystem type %q", fsType)
 		}
 
 		if err != nil {
-			return fmt.Errorf("Could not grow underlying %q filesystem for %q: %s", fsType, devPath, msg)
+			logger.Error("Failed to grow filesystem", logger.Ctx{"filesystem": fsType, "device": devPath, "err": err})
```

Comment:
can we lose the error logging here as we are returning it as an error and it isnt contextual to the volume or instance.